From 0c0ded1d885431bf4509222368278ec6644bf646 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timo=20Ter=C3=A4s?= <timo.teras@iki.fi>
Date: Thu, 16 Apr 2020 15:18:03 +0300
Subject: [PATCH] update-kernel: fallback to old dtbdir if new one does not
 exist

This allows running update-kernel on new alpine and to create
images from stable branch by overriding repository files.

Fixes: 3fd2ce43 "update-kernel: fix dtbs dir"
---
 update-kernel.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/update-kernel.in b/update-kernel.in
index 3d13d25..d9b774f 100644
--- a/update-kernel.in
+++ b/update-kernel.in
@@ -265,6 +265,7 @@ KVER_FLAVOR=
 [ "$FLAVOR" = vanilla ] || KVER_FLAVOR=-$FLAVOR
 KVER=$(basename $(ls -d $ROOT/lib/modules/*"$KVER_FLAVOR"))
 DTBDIR=$ROOT/boot/dtbs-$FLAVOR
+[ -d "$DTBDIR" ] || DTBDIR=$ROOT/usr/lib/linux-$KVER
 depmod -b $ROOT "$KVER"
 
 
-- 
2.26.1

